php/php-pcov/Portfile

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:filetype=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               github 1.0
+PortGroup               php 1.1
+
+name                    php-pcov
+version                 1.0.11
+categories-append       devel
+platforms               darwin
+maintainers             {ryandesign @ryandesign} openmaintainer
+license                 Apache-2
+
+php.branches            7.1 7.2 7.3 7.4 8.0 8.1
+php.pecl                yes
+
+description             A self contained php-code-coverage compatible driver for PHP.
+
+long_description        ${description}
+
+checksums               rmd160  abbf80ccb3883ccfd3a3ce411ede7785ac08bcd6 \
+                        sha256  ad22e64cd3af065330182ac142c1c3ab40de934482cd400f8bd76064130ac242 \
+                        size    25300
+
+use_parallel_build  yes
```

Comment:
this is the default and can, therefore, be removed

Reply:
Aha, amended in fixup commit, thanks!